src/jit/compiler.cpp

```diff
@@ -7862,6 +7863,7 @@ void JitTimer::PrintCsvMethodStats(Compiler* comp)
 
     comp->m_inlineStrategy->DumpCsvData(fp);
 
+    fprintf(fp, "%I64u,", (unsigned __int64)comp->compGetAllocator()->getTotalBytesAllocated());
```

Comment:
Sadly, no. From [MSDN](https://msdn.microsoft.com/en-us/library/tcxf1dw6.aspx): "The hh, j, z, and t length prefixes are not supported." We can use the VC-specific `Iu`, however, and avoid the cast.